### PR TITLE
Remove unused files

### LINKED ISSUE #157: [REFACT] Remove unused files
We removed already old menu so we do not need to have `css/components/pp_adv_menu.css` anymore

### PR DESCRIPTION
**Pull request by AucT**
Closes #157

## Summary by CodeRabbit

- **Chores**
	- Removed several CSS files related to various components, including styles for Bootstrap, advanced menus, lightboxes, carousels, RTL layouts, and reCAPTCHA.
	- This cleanup helps streamline the codebase and may improve performance by eliminating unused styles.

### CODE REVIEW

**Comment by pftg** [comment]
@AucT what about less files? we do not need any files which we do not use

File: themes/beaver/assets/css/components/pp_adv_menu.css
Comment (pftg) [inline themes/beaver/assets/css/components/pp_adv_menu.css:1]: 👍 thanks

**Review by pftg — CHANGES_REQUESTED** [review]
@AucT, there are some unsued or potential unused files: 
jetthoughts github io – 587 js  jetthoughts github io  2024-10-07 14-46-26

jetthoughts github io – bootstrap min js  jetthoughts github io  2024-10-07 14-47-34

**Comment by AucT** [comment]
@pftg 

removed not needed js

**Pull request merged**